packages/liveblocks-core/src/lib/utils.ts

```diff
@@ -208,7 +209,11 @@ export function memoizeOnSuccess<T>(
   return () => {
     if (cached === null) {
       cached = factoryFn().catch((err) => {
-        cached = null;
+        // Keep returning the failed promise for the next 5 seconds, after
+        // which we'll trigger a fresh promise again.
+        setTimeout(() => {
+          cached = null;
+        }, 5_000);
```

Comment:
We will probably (and hopefully) not get to this point (because we auto retry for [5 times](https://github.com/liveblocks/liveblocks/blob/0dbcecfb6eea4ea389e8f4228a753a8ebaaead17/packages/liveblocks-react/src/liveblocks.tsx#L236-L240) before throwing an error), but I would like to hear your thought on this @GuillaumeSalles . 

The implication of doing this, from my understanding, is that if user placed an error boundary above a component that has errored out, resetting the error boundary (until this timeout has completed) will not do anything.

Reply:
> The implication of doing this, from my understanding, is that if user placed an error boundary above a component that has errored out, resetting the error boundary (until this timeout has completed) will not do anything.

This is correct! 👍 

It was either this timeout, or we would have to provide an explicit API to reset this memorized value, which I have tried various forms of, but haven’t found a satisfying solution for that won’t over-complicate things thus far.

I don’t like this timeout, but in practice it works pretty nicely.

Reply:
Explained this a bit more in a code comment for posterity in 7cf046d3c9289aa7ce12ea47b79c95e9b6821961.